.github/workflows/build.yml

```diff
@@ -58,41 +74,70 @@ jobs:
           name: Tests report
           path: 'reports/jest-*.xml'
           reporter: jest-junit
+  
   package:
     name: Package and Push artifacts to Octopus
-    needs: test
+    needs: [build, test]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/download-artifact@v2
         with:
           name: dist
           path: dist
+
       - name: Use Node.js
         uses: actions/setup-node@v2
         with:
           node-version: '16'
           cache: 'npm'
+
       - name: Install Go
         uses: actions/setup-go@v2
         with:
           go-version: '^1.17.7'
+
       - name: Embed octo portable
         run: |
           pwsh ./embed-octo.ps1
+
       - name: Replace versions in tasks and create vsix
         run: |
-          ./pack.ps1 -environment Production -version $Env:OCTOPUS_VERSION -setupTaskDependencies
-          ./pack.ps1 -environment Test -version $Env:OCTOPUS_VERSION
+          ./pack.ps1 -environment Production -version ${{ needs.build.outputs.package_version }} -setupTaskDependencies
+          ./pack.ps1 -environment Test -version ${{ needs.build.outputs.package_version }}
         shell: pwsh
+
       - name: Create Packages
+        id: create-packages
         run: |
-          tar -czf OctoTFS.vsix.$OCTOPUS_VERSION.tar.gz ./dist/Artifacts/**/*.vsix
-          tar -czf OctoTFS.publish.$OCTOPUS_VERSION.tar.gz ./publish.ps1 ./dist/extension-manifest*.json
-      - name: Push Package
-        run: |
-          docker run -e "OCTOPUS_CLI_SERVER=${{ secrets.OCTOPUS_URL }}" -e "OCTOPUS_CLI_API_KEY=${{ secrets.INTEGRATIONS_API_KEY }}" -v $(pwd):/src octopusdeploy/octo push --space "Integrations" --package OctoTFS.vsix.$OCTOPUS_VERSION.tar.gz --package OctoTFS.publish.$OCTOPUS_VERSION.tar.gz --overwrite-mode OverwriteExisting
-      - name: Create Release
+          tar -czf OctoTFS.vsix.${{ needs.build.outputs.package_version }}.tar.gz ./dist/Artifacts/**/*.vsix
+          tar -czf OctoTFS.publish.${{ needs.build.outputs.package_version }}.tar.gz ./publish.ps1 ./dist/extension-manifest*.json
+
+      - name: Install Octopus CLI 🐙
+        uses: OctopusDeploy/install-octopus-cli-action@v1.1.7
```

Comment:
`install-octopus-cli-action` v1.2 is now available

Reply:
Updated 👍